package.json

```diff
@@ -31,5 +31,8 @@
     "mocha": "^8.2.1",
     "puppeteer": "^2.1",
     "webdriverio": "^6.4.2"
+  },
+  "peerDependencies": {
+    "codeceptjs": ">= 2.3.3"
```

Comment:
version chosen based off of whats in the README: https://github.com/codeceptjs/configure#codeceptjs-configuration-hooks-

> Requires CodeceptJS >= 2.3.3